### PR TITLE
Pyladies Munich Update Info

### PR DESCRIPTION
**Pull request by laysauchoa**
Hey all,

I would like to update the info about Munich Pyladies. Please review it :)

### CODE REVIEW

File: munich/README [hunk 1]
```diff
@@ -1,7 +1,19 @@
 ###PyLadies Munich
```
Comment (willingc) [inline munich/README]: I would change this to `# PyLadies Munich`

File: munich/README [hunk 2]
```diff
@@ -1,7 +1,19 @@
...
 
-The repo for PyLadies Munich. 
+We are the Munich chapter of an international mentorship group, PyLadies, with a focus on helping more women become active participants and leaders in the Python open-source community. Our mission is to promote, educate and advance a diverse Python community through outreach, education, conferences, events and social gatherings.
```
Comment (willingc) [inline munich/README]: Style suggestion: Consider splitting this to multiple lines <= 78 characters

File: munich/README [hunk 3]
```diff
@@ -1,7 +1,19 @@
...
-Get social with us! Twitter, via [@pyladiesmunich][1]!
+PyLadies also aims to provide a friendly support network for women and a bridge to the larger Python world. Anyone with an interest in Python is encouraged to participate!
 
-[1]: http://www.twitter.com/pyladiesmunich
+Code of Conduct: https://www.pyladies.com/CodeOfConduct/
+
+http://www.pyladies.com | http://munich.pyladies.com
+
+Follow us:
+MeetUp [@pyladiesmunich][1]!
```
Comment (willingc) [inline munich/README]: I would prefix the Meetup and Facebook lines with `- `. Otherwise will run the lines together

**Comment by willingc** [comment]
I would also copy the logo image file to this location: https://github.com/pyladies/pyladies/tree/master/www/_assets/images

File: munich/README (hunk 3 above)
Comment (laysauchoa) [reply munich/README]: I agree :)

File: munich/README (hunk 2 above)
Comment (laysauchoa) [reply munich/README]: Yes, done :)

File: munich/README (hunk 1 above)
Comment (laysauchoa) [reply munich/README]: done :)

**Comment by willingc** [comment]
Thanks @laysauchoa. The changes are now live :smile:

**Pull request merged**